app/controllers/orders_controller.rb

```diff
@@ -1,5 +1,28 @@
 class OrdersController < ApplicationController
+  before_action :logged_in_user, only: [:show, :edit, :update, :destroy]
+  include SessionsHelper
+
+  def create
+    order_number = p SecureRandom.hex(2)
+    @order = Order.create(user_id: current_user.id, order_number: order_number)
+    @cart_items = current_user.cart.cart_items
+      @cart_items.each do |cart_item|
```

Comment:
`@cart_items`を他の箇所で使用することはなそうなので、インスタンス変数である必要ないです！
ローカル変数で良いです、というより、直接each文にくっつけてしまっても良いと思います！
（人によっては変数に一回入れてあげるだけで可読性が上がるので好き、となるかもしれないので、森さんが変数に入れてあげたいと感じる場合はローカル変数に入れても構いません！）
```suggestion
      current_user.cart.cart_items.each do |cart_item|
```

Reply:
上記対応いたしました。
https://github.com/thimon21/railsthimon/pull/11/commits/b9ef827d7fd1abf78fd748304e98271b061dfd31